src/useValidation/useValidation.ts

```diff
@@ -61,10 +61,10 @@ function isRules<T>(value: MaybeRef<string> | RulesArg<T>): value is RulesArg<T>
   return typeof unref(value) !== 'string'
 }
 
-export function useValidation<T>(value: MaybeRef<T>, rules: RulesArg<NoInfer<T>>): UseValidation
-export function useValidation<T>(value: MaybeRef<T>, name: MaybeRef<string>, rules: RulesArg<NoInfer<T>>): UseValidation
+export function useValidation<T>(value: MaybeRefOrGetter<T>, rules: RulesArg<NoInfer<T>>): UseValidation
+export function useValidation<T>(value: MaybeRefOrGetter<T>, name: MaybeRef<string>, rules: RulesArg<NoInfer<T>>): UseValidation
 export function useValidation<T>(
-  value: MaybeRef<T>,
+  value: MaybeRefOrGetter<T>,
   nameOrRules: MaybeRef<string> | RulesArg<NoInfer<T>>,
```

Comment:
I looked into trying to make name be passable via a getter as well, but it gets more complicated trying to distinguish between a rule (which can be a function) and a getter.

Reply:
yeah probably not worth it right now.